src/Symfony/Bridge/Twig/Extension/IntlExtension.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Extension;
+
+use Symfony\Component\Intl\DateIntervalFormatter\DateIntervalFormatter;
+use Symfony\Component\Intl\DateIntervalFormatter\DateIntervalFormatterInterface;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+class IntlExtension extends AbstractExtension
```

Comment:
What about doing this in `https://github.com/twigphp/intl-extra/blob/2.x/IntlExtension.php` instead?
Currently it may conflict with https://github.com/twigphp/twig-extra-bundle/blob/2.x/Resources/config/intl.php.